scss/components/_award.scss

```diff
@@ -79,3 +85,23 @@
   min-height: 10px;
   min-width: 10px;
 }
+
+.civiawards__award__review-panel__to-label {
+  float: left;
+  line-height: 28px;
```

Comment:
Why do we need this non standard line height here instead of using `$line-height` variable from shoreditch?

Reply:
Fixed it, using `$crm-gap-small`.